CONTRIBUTING.md

```diff
@@ -45,6 +45,11 @@ What constitutes a business day is managed by the gem `business_time` and is con
 
 ### App setup
 
+Before running bin/setup, ensure that 'foreman' is removed from the Gemfile.  
```

Comment:
ooooh what happens if it is already installed? Maybe we should update setup script to play more nicely with the Gemfile as is...